src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/BookingCaseNotesResourceIntTest.kt

```diff
@@ -0,0 +1,311 @@
+package uk.gov.justice.hmpps.prison.api.resource.impl
+
+import org.apache.commons.lang3.StringUtils
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.DisplayName
+import org.junit.jupiter.api.Nested
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.springframework.beans.factory.annotation.Autowired
+import org.springframework.http.MediaType
+import org.springframework.web.reactive.function.BodyInserters
+import uk.gov.justice.hmpps.prison.api.model.CaseNote
+import uk.gov.justice.hmpps.prison.api.model.NewCaseNote
+import uk.gov.justice.hmpps.prison.repository.jpa.repository.OffenderCaseNoteRepository
+import uk.gov.justice.hmpps.prison.util.DateTimeConverter
+import java.time.ZoneOffset
+
+class BookingCaseNotesResourceIntTest : ResourceTest() {
+
+  @Autowired
+  lateinit var offenderCaseNoteRepository: OffenderCaseNoteRepository
+
+  @Nested
+  @DisplayName("GET /api/bookings/{bookingId}/caseNotes")
+  inner class GetCaseNotes {
+    @Test
+    fun testCanRetrieveCaseNotes() {
+      webTestClient.get()
+        .uri("/api/bookings/-2/caseNotes")
+        .headers(setAuthorisation("ITAG_USER", listOf("")))
+        .exchange()
+        .expectStatus().isOk
+        .expectBody()
+        .json("case_notes_offender_1.json".readFile())
```

Comment:
Has this been converted from Java? Just wondering if this would be a good opportunity to to switch to `jsonPath` instead of comparing with a json file?

Reply:
It has been converted from java - and I agree - this should be updated too to pull out the bits we want to check